File: JavaScript/7js 2/printerError.js
/*
|--------------------------------------------------------------------------
| Printer Errors
|--------------------------------------------------------------------------
|
*/

function printerError(s) {
   res = 0
   x = 'abcdefghijklm'.split('')
   s.split('').forEach(i => {
      if (!x.includes(i)) res += 1
   });
   return `${res}/${s.length}`
}


function printerError(s) {
   return `${s.replace(/[a-m]/gi, "").length}/${s.length}`;
}


function printerError(s) {
   count = 0;
   for (i = 0; i < s.length; i++) {
      if (s[i] > "m") count++
   }
   return count + "/" + s.length;
}


function printerError(s) {
   return (s.match(/[n-z]/g) || []).length + '/' + s.length
}


console.log(printerError('aaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbmmmmmmmmmmmmmmmmmmmxyz'), "3/56")

File: Python/7py 2/printer_error.py
"""
|--------------------------------------------------------------------------
| Printer Errors
|--------------------------------------------------------------------------
|
"""

def printer_error(s):
   x = list('abcdefghijklm')
   return f'{len([i for i in s if i not in x])}/{len(s)}'


def printer_error(s):
   return "{}/{}".format(len([x for x in s if x not in "abcdefghijklm"]), len(s))


def printer_error(s):
   return '{}/{}'.format(sum(color > 'm' for color in s), len(s))


def printer_error(s):
   return "%s/%s" % (len(s.translate(None, "abcdefghijklm")), len(s))
   

print(printer_error('aaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbmmmmmmmmmmmmmmmmmmmxyz'), "3/56")